CHANGELOG.rst

```diff
@@ -2,6 +2,18 @@
 Changelog for package geometric_shapes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.1.1 (2021-12-22)
+------------------
+* Fix cmake such that Boost::filesystem is exported properly (`#206 <https://github.com/ros-planning/geometric_shapes/issues/206>`_) (`#211 <https://github.com/ros-planning/geometric_shapes/issues/211>`_)
+  Co-authored-by: Jafar Abdi <cafer.abdi@gmail.com>
```

Comment:
Co-authors can be dropped here and listed under "Contributors", imo. Also, do we want to keep/drop CI changes from the changelog in the future?